Demo/src/main/java/com/paypal/android/ui/paypal/PayPalButtonsFragment.kt

```diff
@@ -0,0 +1,184 @@
+package com.paypal.android.ui.paypal
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.AdapterView
+import android.widget.ArrayAdapter
+import android.widget.LinearLayout
+import androidx.appcompat.widget.LinearLayoutCompat
+import androidx.fragment.app.Fragment
+import com.paypal.android.databinding.FragmentPaypalButtonsStylingBinding
+import com.paypal.android.ui.paymentbutton.*
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class PayPalButtonsFragment : Fragment() {
+
+    private val selectedButtonFundingType: ButtonFundingType
+        get() {
+            return ButtonFundingType.values()[
+                    binding.paymentButtonFundingSpinner.selectedItemPosition
+            ]
+        }
+
+    private val selectedColor: PaymentButtonColor?
+        get() = when (selectedButtonFundingType) {
+            ButtonFundingType.PAYPAL_CREDIT ->
+                binding.paymentButtonColorSpinner.selectedItem as? PayPalCreditButtonColor
+            else ->
+                binding.paymentButtonColorSpinner.selectedItem as? PayPalButtonColor
+        }
+
+    private val selectedLabel: PayPalButtonLabel?
+        get() = when (selectedButtonFundingType) {
+            ButtonFundingType.PAYPAL -> PayPalButtonLabel.valueOf(
+                binding.paymentButtonLabelSpinner.selectedItem.toString()
+            )
+            else -> null
+        }
+
+    private val selectedShape: PaymentButtonShape
+        get() = PaymentButtonShape.valueOf(
+            binding.paymentButtonShapeSpinner.selectedItem.toString()
+        )
+
+    private val selectedSize: PaymentButtonSize
+        get() = PaymentButtonSize.valueOf(
+            binding.paymentButtonSizeSpinner.selectedItem.toString()
+        )
+
+    private val binding: FragmentPaypalButtonsStylingBinding by lazy {
+        FragmentPaypalButtonsStylingBinding.inflate(layoutInflater)
```

Comment:
🧶  : I'm looking at [this](https://developer.android.com/topic/libraries/data-binding/generated-binding) on Google's developer site:
![Screen Shot 2022-05-09 at 4 39 48 PM](https://user-images.githubusercontent.com/58225613/167503243-59b63174-f670-42a2-b257-0ff57f3c0a58.png)

We should prefer the `FragmentPaypalButtonsStylingBinding.inflate(layoutInflater, viewGroup, false)` inflate variant here to be safe.

Reply:
@sshropshire this is needed, when we're attaching to parent, in this case we're not doing, so either method will have same impact, used this one because it can be put separately in val rather than in `onCreate()`

Reply:
Interesting. Is this a pattern y'all had on PP credit? I'm not opposed, just looking to understand. I've never seen inflation without specifying the view group and whether or not to attach to parent.

The template (without view binding) generated in Android projects uses [this method](https://developer.android.com/reference/android/view/LayoutInflater#inflate(int,%20android.view.ViewGroup,%20boolean)) and honestly just feels right since the method signatures line up. Do you feel there's a specific advantage to having it in a `val`?